CMakeLists.txt

```diff
@@ -253,7 +253,11 @@ endif()
 if(EXISTS ${PCI_IDS_GZ_PATH} AND NOT EXISTS ${CMAKE_STAGING_PREFIX}/${PCI_IDS_INSTALL_PATH})
     message(STATUS "Extracting PCI ID database from ${PCI_IDS_GZ_PATH}...")
     file(MAKE_DIRECTORY "${CMAKE_STAGING_PREFIX}/${CMAKE_INSTALL_DATAROOTDIR}")
-    execute_process(COMMAND "${GZIP_TOOL}" -d -c "${PCI_IDS_GZ_PATH}" OUTPUT_FILE "${CMAKE_STAGING_PREFIX}/${PCI_IDS_INSTALL_PATH}")
+    if (CMAKE_VERSION VERSION_LESS 3.18.0)
```

Comment:
The commit message needs updated to reflect that we aren't bumping the minimum required yet.

Reply:
Done, see [eab454f](https://github.com/SerenityOS/serenity/pull/15040/commits/eab454f867f88b061e4d2d3ad9797cff7db037b5).